kleva-j/iReporter
# Feature/#162527728/update an incident record using jwt

#### What does this PR do?
- Ensures an authenticated user can update their records

#### Description of the task to be completed
 - create functions to authenticate a user credentials
 - create functions to handle requests to update a red-flag/intervention record's location
 - create functions to handle requests to update a red-flag/intervention record's comment
 - create functions to handle requests to update a red-flag/intervention record's status

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#162503997/update-an-incident-record-using-jwt
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162527728](https://www.pivotaltracker.com/story/show/162527728)
